src/translations/nbTranslations.json

```diff
@@ -1120,22 +1120,23 @@
   },
   "project": {
     "add_affiliation": "Legg til tilknytning",
+    "add_project_manager": "Legg til prosjektleder",
     "add_project_contributor": "Legg til prosjektdeltaker",
     "affiliation_modal": {
       "add_new_affiliation_helper_text": "<0>Bare institusjoner som prosjektet er gjennomført i samarbeid med, skal legges til som tilknytning til prosjektdeltaker.</0> <0>Legg til tilknytning til prosjektdeltaker ved å først søke opp institusjonen og eventuelt enhet.</0> <0>Tilhører prosjektdeltakeren flere forskjellige enheter under samme institusjon, må du legge til dette som ny tilknytning.</0>",
-      "cannot_delete_affiliation_for_project_manager": "Kan ikke fjerne tilknytning for prosjektleder",
+      "cannot_delete_affiliation_for_project_manager": "Prosjektleder må ha en tilknytning",
       "cannot_delete_only_affiliation": "Kan ikke fjerne eneste tilknytning",
       "delete_affiliation": "Fjern tilknytning",
       "edit_affiliation": "Endre tilknytning"
     },
+    "as_project_manager": "som prosjektleder",
```

Comment:
liten heads up her også, med at oversettere ikke har noen enkel måte å se hvordan denne vil brukes, så de vil tipper de fort vil legge til stor forbokstav for oversettelser her.
Kan være tryggere å ha en dedikert key til det, feks: `"remove_project_manager": "Vil du fjerne {{name}}som prosjektleder?"`